### PR TITLE
fix range offset

### PR DESCRIPTION
**Pull request by SleimanJneidi**

### CODE REVIEW

File: src/main/java/com/iabtcf/decoder/TCString.java [hunk 1]
```diff
@@ -27,7 +27,7 @@
 import com.iabtcf.v2.PublisherRestriction;
 
 public interface TCString {
-    public static TCString decode(String consentString) {
+    static TCString decode(String consentString) {
```
Comment (laktech) [inline src/main/java/com/iabtcf/decoder/TCString.java]: this needs to be public

File: src/test/java/com/iabtcf/decoder/TCStringV2Test.java [hunk 2]
```diff
@@ -117,6 +118,14 @@ public void testCanParseAllParts() {
         assertEquals(1, IntIterableUtils.toStream(tcModel.getPubPurposesConsent()).count());
     }
 
+    @Test
+    public void testCanParseRangeEncodedVendorLegitimateInterests() {
+        String base64CoreString =
+                "COv__-wOv__-wC2AAAENAPCgAAAAAAAAAAAAA_wAQA_gEBABAEAAAA";
+        TCString tcModel = parse(base64CoreString);
+        assertEquals(new TreeSet<>(Arrays.asList(128)), IntIterableUtils.toSet(tcModel.getVendorLegitimateInterest()));
```
Comment (laktech) [inline src/test/java/com/iabtcf/decoder/TCStringV2Test.java]: you can use,

import static com.iabtcf.test.utils.IntIterableMatcher.matchInts;

assertThat(tcModel.getVendorLegitimateInterest(), matchInts(128));

File: src/main/java/com/iabtcf/decoder/TCString.java (hunk 1 above)
Comment (SleimanJneidi) [reply src/main/java/com/iabtcf/decoder/TCString.java]: public is redundant for interface methods

File: src/test/java/com/iabtcf/decoder/TCStringV2Test.java (hunk 2 above)
Comment (SleimanJneidi) [reply src/test/java/com/iabtcf/decoder/TCStringV2Test.java]: fixed

**Comment by laktech** [comment]
still getting,

[INFO] There is 1 error reported by Checkstyle 7.1.2 with checkstyle.xml ruleset.
[ERROR] src/test/java/com/iabtcf/decoder/TCStringV2Test.java:[33,8] (imports) UnusedImports: Unused import - java.util.TreeSet.

**Comment by SleimanJneidi** [comment]
@laktech fixed

**Pull request merged**